src/scss/components/_mobile-scroll-carousel.scss

```diff
@@ -0,0 +1,25 @@
+/**
+ * Mobile scroll carousel
+ * --------------------------------
+ * Used for mobile filter carousel
+ *
+ * ================================= */
+
+.c-mobileScrollCarousel {
+
+    @include media('<mid') {
+        overflow-x: scroll;
+        margin: 0 (spacing(x2) * -1);
+        -webkit-overflow-scrolling: touch;
```

Comment:
Does this not get taken care of by auto-prefixer?

Reply:
`-webkit-overflow-scrolling: touch;` is not autoprefixed, unfortunately. It's to give mobiles "momentum" scrolling on overflow-scroll :)